sass/public.scss

```diff
@@ -0,0 +1,235 @@
+@import '/sass/variables.scss';
+
+/**********************************/
+
+
+/*********** Nav Bar CSS **********/
+
+
+/**********************************/
+
+.navbar {
+    position: relative;
+    transition: .5s;
+    z-index: 999;
```

Comment:
let's create a zindex map and use it here this way we can keep track of the z index values that we have